src/chai/managed_ptr.hpp

```diff
@@ -153,7 +133,7 @@ namespace chai {
    ///       only be valid in the correct context. Take care when passing raw pointers
    ///       as arguments to member functions.
    ///    Be aware that CHAI checks every CUDA API call for GPU errors by default. To
-   ///       turn off GPU error checking, pass -DCHAI_ENABLE_GPU_ERROR_CHECKING=OFF as
+   ///       turn off GPU error checking, pass -DCHAI_ENABLE_CHAI_GPU_ERROR_CHECKING=OFF as
```

Comment:
I think this shouldn't have been changed.

Reply:
this is now fixed